lisa/transformers/upgrade_packages.py

```diff
@@ -16,12 +16,12 @@
 @dataclass
 class UpgradeInstallerSchema(schema.TypedSchema, schema.ExtendableSchemaMixin):
     repo_url: str = field(
-        default="http://azure.archive.ubuntu.com/ubuntu/",
+        default="",
```

Comment:
The repo url can be here for default behavior. It's not easy to use, for each required value. Unless this value is optional. But it looks not.

Reply:
If it's not a required value, please remove the below `metadata=field_metadata(required=True),`

Reply:
Resolved